README.md

```diff
@@ -1,45 +1,54 @@
+Snowplow is the world’s largest developer-first engine for collecting behavioral data. In short, it allows you to:
 
-**To find out more, please check out the [Snowplow website][website] and the [docs website][docs].**
+* Collect events such as impressions, clicks, video playback (or even custom events of your choosing).
+* Store the data in a scalable data warehouse you control ([Amazon Redshift](https://aws.amazon.com/redshift/), [Databricks](https://databricks.com/product/databricks-sql), [Elasticsearch](https://www.elastic.co/), [Google BigQuery](https://cloud.google.com/bigquery), [Snowflake](https://www.snowflake.com/workloads/data-warehouse-modernization/)) or emit it via a stream ([Amazon Kinesis](https://aws.amazon.com/kinesis/), [Google PubSub](https://cloud.google.com/pubsub/docs/overview)).
+* Leverage the biggest range of tools to model and analyze the behavioral data: [dbt](https://www.getdbt.com/), [Looker](https://www.looker.com/), [Metabase](https://www.metabase.com/), [Mode](https://mode.com/), [Superset](https://superset.apache.org/), [Redash](https://redash.io/), and more.
 
-### Version Compatibility Matrix
-
-For compatibility assurance, the version compatibility matrix offers clarity on our recommended stack. It is strongly recommended when setting up a Snowplow pipeline to use the versions listed in the version compatibility matrix which can be found [within our docs][version-compatibility].
+Thousands of organizations of all sizes around the world generate, enhance, and model behavioral data with Snowplow to fuel [advanced analytics](https://snowplowanalytics.com/advanced-analytics/?utm_source=github&utm_content=main-repo), [AI/ML initiatives](https://snowplowanalytics.com/ai-ml/?utm_source=github&utm_content=main-repo), or [composable CDPs](https://snowplowanalytics.com/composable-cdp/?utm_source=github&utm_content=main-repo).
 
-### Public Roadmap
+### Table of contents
 
-This repository also contains the [Snowplow Public Roadmap][roadmap]. The Public Roadmap lets you stay up to date and find out what's happening on the Snowplow Platform. Help us prioritize our cards: open the issue and leave a 👍 to vote for your favorites. Want us to build a feature or function? Tell us by heading to our [Discourse forum][discourse] 💬.
+* [Why Snowplow?](#why-snowplow)
+* [Where to start?](#-where-to-start-%EF%B8%8F)
+* [Snowplow technology 101](#snowplow-technology-101)
+* [Version compatibility matrix](#version-compatibility-matrix)
+* [About this umbrella repository](#about-this-repository)
+* [Public roadmap](#public-roadmap)
+* [Community](#community)
 
-### Try Snowplow
+### Why Snowplow?
 
-Setting up a full open-source Snowplow pipeline requires a non-trivial amount of engineering expertise and time investment.
-You might be interested in finding out what Snowplow can do first, by setting up [Try Snowplow][try-snowplow].
+* 🚀 **Battle-tested architecture** capable of processing billions of events per day.
```

Comment:
Can we change this to not be `Battle-tested`, particularly with the 🚀 emoji? 😬

Perhaps: Robust architecture? Tried and tested architecture?